packages/waku/src/lib/builder/serve-cloudflare.ts

```diff
@@ -1,41 +1,75 @@
 import { Hono } from 'hono';
+import { unstable_getPlatformObject } from 'waku/server';
 import { runner } from '../hono/runner.js';
+import type {
+  ExportedHandler,
+  fetch,
```

Comment:
This is confusing. So:
```suggestion
  fetch as CloudflareFetch,
```